src/main/java/teammates/ui/webapi/action/BasicFeedbackSubmissionAction.java

```diff
@@ -126,7 +126,7 @@ protected void checkAccessControlForInstructorFeedbackSubmission(
         } else {
             gateKeeper.verifyLoggedInUserPrivileges();
             gateKeeper.verifyAccessible(logic.getInstructorForGoogleId(feedbackSession.getCourseId(), userInfo.getId()),
-                    feedbackSession, Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
+                    feedbackSession, Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS);
```

Comment:
The previous setting `INSTRUCTOR_PERMISSION_MODIFY_SESSION` is correct for previewing. This is correct for moderation only. Refer to `checkAccessControlForStudentFeedbackSubmission`.